src/components/Main/Greetings.style.js

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import {
+  bold24,
+  bold18,
+  reg18,
+  m_bold24,
+  m_bold18,
+  m_reg18,
+} from '../../styles/font';
+
+const S = {
+  GreetingsLayout: styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    gap: 7.75rem;
+    @media (max-width: 1279px) {
+      flex-direction: column;
+      align-items: center;
+      gap: 1.5rem;
+    }
+  `,
+  PhotoWrapper: styled.img`
+    width: 16.1875rem;
+    height: 21.5833rem;
+    border-radius: 1rem;
+    background: lightgray 50% / cover no-repeat;
+    @media (max-width: 1279px) {
+      width: 20.9375rem;
+      height: 28.25rem;
+    }
+  `,
+  MemberContainer: styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
```suggestion
    @media (max-width: 1279px) {
      align-items: center;
    }
```
이렇게 수정해야 모바일에서만 중앙정렬 될 듯 합니다

Reply:
앗 깜빡했네요 수정하겠습니다!